win32ss/user/user32/windows/class.c

```diff
@@ -1856,7 +1856,7 @@ SetWindowLongW(
   int nIndex,
   LONG dwNewLong)
 {
-    return NtUserSetWindowLong(hWnd, nIndex, dwNewLong, FALSE);
+    return (LONG)NtUserSetWindowLongPtr(hWnd, nIndex, dwNewLong, FALSE);
```

Comment:
I'm not sure this is useful to do those three changes, since it is OK to manipulate those WORD/LONG with the NtUserSetWindowLong (not Ptr) function even in x64.

Reply:
It's for code uniformity.